components/date-picker/date-picker.examples.js

```diff
@@ -49,6 +49,40 @@ singleDate.story = {
   name: 'single date'
 };
 
+export const singleDateAndTime = () => {
+  class DatePickerExample extends Component {
+    state = {
+      date: '01.01.18',
+      time: '9:45'
+    };
+
+    setDate = ({date, time}) => {
+      this.setState({date, time});
+    };
+
+    render() {
+      return (
+        <div>
+          <DatePicker
+            date={this.state.date}
+            time={this.state.time}
+            onChange={this.setDate}
+            withTime
+            clear
+          />
+        </div>
+      );
+    }
+  }
+
+  return <DatePickerExample/>;
+};
+
+singleDateAndTime.story = {
+  hermione: {skip: true},
```

Comment:
Sorry, I gave you a wrong code sample: it should be `parameters: {hermione: {skip: true}}`

Reply:
will fix

Reply:
done